xsd/siri_utility/siri_location-v2.0.xsd

```diff
@@ -141,31 +145,27 @@ Rail transport, Roads and road transport
    <xsd:documentation>GML Spatial coordinate reference system.</xsd:documentation>
   </xsd:annotation>
  </xsd:element>
- <!--====== LOCATIONe ============================================================-->
+ <!--====== LOCATION ============================================================-->
  <xsd:complexType name="LocationStructure">
   <xsd:annotation>
    <xsd:documentation>Type for gepspatial Position of a point. May be expressed in concrete WGS 84 Coordinates or any gml compatible point coordinates format.</xsd:documentation>
   </xsd:annotation>
   <xsd:sequence>
    <xsd:choice>
     <xsd:sequence>
-     <xsd:element name="Longitude" type="LongitudeType">
-      <xsd:annotation>
-       <xsd:documentation>Longitude from Greenwich Meridian. -180 (West) to +180 (East). Decimal degrees. eg 2.356</xsd:documentation>
-      </xsd:annotation>
-     </xsd:element>
-     <xsd:element name="Latitude" type="LatitudeType">
-      <xsd:annotation>
-       <xsd:documentation>Latitude from equator. -90 (South) to +90 (North). Decimal degrees. eg 56.356</xsd:documentation>
-      </xsd:annotation>
-     </xsd:element>
+     <xsd:group ref="WgsGroup"/>
     </xsd:sequence>
     <xsd:element name="Coordinates" type="CoordinatesStructure">
      <xsd:annotation>
       <xsd:documentation>Coordinates of points in a GML compatibe format, as indicated by srsName attribute.</xsd:documentation>
      </xsd:annotation>
     </xsd:element>
    </xsd:choice>
+   <xsd:element name="Radius" type="DistanceType" minOccurs="0">
```

Comment:
The LocationStructure is used for quite a few different 'points', most of which do not support a Radius. For example, the corners of a BoundingBox.
Therefore I would NOT put Radius as part of LocationStructure, but add it specifically as a separate attribute (next to Location) where it is needed.

Reply:
See Commit [abd9a81](https://github.com/SIRI-CEN/SIRI/pull/64/commits/abd9a81d05b94287158fe8d4e28029db2475e4de)